txhelpers/subsidy_test.go

```diff
@@ -13,23 +13,23 @@ import (
 func TestUltimateSubsidy(t *testing.T) {
 	// Mainnet
 	wantMainnetSubsidy := int64(2099999999800912)
-	totalSubsidy := UltimateSubsidy(chaincfg.MainNetParams(), -1)
+	totalSubsidy := UltimateSubsidy(chaincfg.MainNetParams(), -1, -1)
 
 	if totalSubsidy != wantMainnetSubsidy {
 		t.Errorf("Bad total subsidy; want %d, got %d",
 			wantMainnetSubsidy, totalSubsidy)
 	}
 
 	// verify cache
-	totalSubsidy2 := UltimateSubsidy(chaincfg.MainNetParams(), -1)
+	totalSubsidy2 := UltimateSubsidy(chaincfg.MainNetParams(), -1, -1)
 	if totalSubsidy != totalSubsidy2 {
 		t.Errorf("Bad total subsidy; want %d, got %d",
 			totalSubsidy, totalSubsidy2)
 	}
 
 	// Mainnet with dcp0010 activating at 638976
 	wantMainnetSubsidy = int64(2100000000015952)
-	totalSubsidyDCP0010 := UltimateSubsidy(chaincfg.MainNetParams(), 638976)
+	totalSubsidyDCP0010 := UltimateSubsidy(chaincfg.MainNetParams(), 638976, -1)
```

Comment:
Perhaps add a test for totalSubsidyDCP0012?

Relevant bits from dcrd tests (although the estimated activation height used is a bit earlier than it will end up being -- it was an estimate at the time after all!):

```Go
	const dcp0010ActivationHeight = 657280
	const estimatedDCP0012ActivationHeight = 782208
	const expectedTotalSubsidy = 2099999998387408
```

Alternatively, you might want to wait until we know the final real activation height and update it here accordingly.  I will likely update the dcrd tests once we know the real activation height (which at this point I think will likely be 794368, but we'll see).

Reply:
I was actually planning to ask what that height might be and what value you had.  Will start with the most optimistic guess

Reply:
So, if it ends up being ` 794368` as I expect based on the current info, it'd be:

```Go
	const dcp0010ActivationHeight = 657280
	const estimatedDCP0012ActivationHeight = 794368
	const expectedTotalSubsidy = 2099999998394320
```